warn/stacktrace.go

```diff
@@ -0,0 +1,65 @@
+package warn
```

Comment:
The package needs a comment.
Please explain what it is and how it's used.

Reply:
Obsolete (removed the file together with stacktraces).